ext/spl/spl_directory.stub.php

```diff
@@ -90,7 +90,7 @@ public function setFileClass(string $class_name = SplFileObject::class) {}
     /** @return void */
     public function setInfoClass(string $class_name = SplFileInfo::class) {}
 
-    /** @alias SplFileInfo::getPathname */
+    /** @implementation-alias SplFileInfo::getPathname */
```

Comment:
As far as I saw the manual doesn't report them as aliases, that's the only reason why I labeled them as implementation-aliases. I think we should just make them regular aliases, which would make the heuristics mentioned in the description still hold. There are a couple of similar cases in ext/spl where the manual doesn't report a magic method as alias (sometimes there is a note though).